### DISCUSSION TITLE
YOLO v8 question?

### DISCUSSION THREAD

**Discussion by LeMinhMan2809** [ultralytics/ultralytics#3263] [Docs]
Hello everyone, I am researching YOLOv8. Does YOLOv8 have a published paper and network architecture about the v8 version?

**Comment by pderrenger** [comment]
@LeMinhMan2809 hello! YOLOv8 does not have a dedicated academic paper. The YOLO series of models, including YOLOv8, are continuously updated with improvements in the Ultralytics repository. For detailed information on the network architecture and updates, you can refer to the official documentation and the model configuration files in the repository. The documentation provides comprehensive guidance on the architecture, usage, and features of YOLOv8. 📚🚀
